lib/sort.py

```diff
@@ -11,11 +11,13 @@ def default_sort(rows, top_score_metric):
 
 def pvacbind_sort(rows, top_score_metric):
     if top_score_metric == 'median':
-        sorted_rows = sorted(rows, key=lambda row: (float(row['Best Score'])))
+        sorted_rows = sorted(rows, key=lambda row: (row['Mutation']))
+        sorted_rows = sorted(sorted_rows, key=lambda row: (float(row['Best Score'])))
         sorted_rows = sorted(sorted_rows, key=lambda row: (int(row['Sub-peptide Position'])))
         sorted_rows = sorted(sorted_rows, key=lambda row: (float(row['Median Score'])))
```

Comment:
Would it make sense to combine these into a single sort? Something like:
```python
sorted_rows = sorted(rows, key=lambda row: (float(row['Median Score']), int(row['Sub-peptide Position']), float(row['Best Score']), row['Mutation']))
```

Reply:
sure thing. Just pushed a commit with that update.